tools/topology/sof/pipe-smart-amplifier-playback.m4

```diff
@@ -69,6 +69,15 @@ C_CONTROLBYTES(Smart_amp Model, PIPELINE_ID,
         ,
         MODEL_priv)
 
+
+C_CONTROLBYTES_VOLATILE_READONLY(Smart_amp Model_Get_params, PIPELINE_ID,
```

Comment:
@lgirdwood this can only be added if the ABI version is >= 3.17 isnt it?

Reply:
yes, the control type tuple ID is ABI 3.17

Reply:
@RDharageswari can you please make that change here?

Reply:
Done